### ISSUE TITLE
Color Input: Eyedropper can't be exited via keyboard

### ISSUE THREAD

**Issue by samwhale** [swissspidy/web-stories-wp#11089]
## Bug Description

The eyedropper may be opened via keyboard (in the design panel) but the eyedropper cannot be closed/exited with keyboard.

## Expected Behaviour

A user should be able to exit out of eyedropper mode with keyboard. This could be done with the `esc` key.

## Steps to Reproduce

1. Use the eyedropper in the design panel or floating menu via keyboard
2. Try to exit the eyedropper state with keyboard. `esc` doesn't work, and `tab` will shift the focus around while in eyedropper mode.

## Screenshots

![keyboard](https://user-images.githubusercontent.com/22185279/160172836-f02d4fa4-e2e1-41ec-b30c-bb170df2cf5c.gif)

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

**Comment by swissspidy** [comment]
Duplicate of #11051
